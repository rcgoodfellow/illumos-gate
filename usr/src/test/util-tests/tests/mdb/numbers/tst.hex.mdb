0x123456789abcdef=K
0x123____456_789___abcd__ef=K
0x1_2_3_4_5_6_7_8_9_a_b_c_d_e_f____=K
0t16$d
badcafe=K
bad___cafe=K
bad___ca_fe=E
