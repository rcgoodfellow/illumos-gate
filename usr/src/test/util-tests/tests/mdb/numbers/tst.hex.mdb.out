                123456789abcdef 
                123456789abcdef 
                123456789abcdef 
radix = 16 base ten
                badcafe         
                badcafe         
                195939070            
